.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+version: 2
+jobs:
+  test:
+    docker:
+      - image: docker:stable-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          command: docker build -t mps .
+      - run:
+          name: Test Code
+          command: docker run mps
+  integrate:
+    docker:
+      - image: mozilla/edge-validator:latest
```

Comment:
Should we use a stable tag?

Reply:
Not a bad idea, considering some of the potential changes.